features/provision.feature

```diff
@@ -45,6 +45,29 @@ Feature: Provisioning
       }
       """
 
+
+  @conjur-version-5
+  Scenario: Provision resource with service broker API 2.15 or better
+    When I PUT "/v2/service_instances/9b292a9c-af66-4797-8d98-b30801f32ax7" with body:
+    """
+    {
+      "context": {
+        "organization_guid": "e027f3f6-80fe-4d22-9374-da23a035ba0a",
+        "space_guid": "8c56f85c-c16e-4158-be79-5dac74f970db"
+        "organization_name": "my-organization",
+        "space_name": "my-space"
+      },
+      "service_id": "c024e536-6dc4-45c6-8a53-127e7f8275ab",
+      "plan_id": "3a116ac2-fc8b-496f-a715-e9a1b205d05c.community",
+      "organization_guid": "e027f3f6-80fe-4d22-9374-da23a035ba0a",
+      "space_guid": "8c56f85c-c16e-4158-be79-5dac74f970db",
+      "parameters": {
+      }
+    }
+    """
+    Then the HTTP response status code is "201"
```

Comment:
This test is failing. It is expecting 201, but getting 400. Also seeing the below error.
Then the HTTP response status code is "400"                                        # features/step_definitions/response_steps.rb:1

[2021-06-03T19:01:33.374Z]     And the JSON should be {}                                                          # json_spec-1.1.5/lib/json_spec/cucumber.rb:29

[2021-06-03T19:01:33.374Z]       Expected equivalent JSON

[2021-06-03T19:01:33.374Z]       Diff:

[2021-06-03T19:01:33.374Z]       @@ -1,295 +1,589 @@

[2021-06-03T19:01:33.374Z]        {

[2021-06-03T19:01:33.374Z]       +  "error": "Bad Request",

[2021-06-03T19:01:33.375Z]       +  "exception": "#<ActionDispatch::Http::Parameters::ParseError: 765: unexpected token at '{\n  \"context\": {\n    \"organization_guid\": \"e027f3f6-80fe-4d22-9374-da23a035ba0a\",\n    \"space_guid\": \"8c56f85c-c16e-4158-be79-5dac74f970db\"\n    \"organization_name\": \"my-organization\",\n    \"space_name\": \"my-space\"\n  },\n  \"service_id\": \"c024e536-6dc4-45c6-8a53-127e7f8275ab\",\n  \"plan_id\": \"3a116ac2-fc8b-496f-a715-e9a1b205d05c.community\",\n  \"organization_guid\": \"e027f3f6-80fe-4d22-9374-da23a035ba0a\",\n  \"space_guid\": \"8c56f85c-c16e-4158-be79-5dac74f970db\",\n  \"parameters\": {\n  }\n}'>",

[2021-06-03T19:01:33.375Z]       +  "status": 400,

Reply:
Do we have a cause? not sure these logs tell us the issue with the code.

Reply:
found it. was invalid JSON, no comma after ```space_guid```. fixed!